assignment1.rb

```diff
@@ -28,14 +28,13 @@ def sum_of_cubes(a, b)
 
 
 def non_duplicated_values(arr)
-    counts = Hash.new(0)
-
+    res = Array.new
     arr.each do |ele|
-        counts[ele]+=1
+        if arr.count(ele) == 1
+            res.push(ele)
+        end
     end
-    counts.select do |key, count|
-        count == 1
-    end.keys
+    return res
```

Comment:
Still you can simplify..

Reply:
I tried my best but can't able to find solution to simplify this further.. can you please mention that method or share the link of that method

Reply:
arr = [1,2,2,3,3,4,5]
arr.select {|el| arr.count(el).eql?(1)}